recipe/meta.yaml

```diff
@@ -21,23 +22,31 @@ requirements:
     - wheel
   run:
     - python
-    - matplotlib-base >=3.2
-    - numpy >=1.21.0
+    - matplotlib-base >=3.5
+    - numpy >=1.22.0,<2.0
     - scipy >=1.8.0
-    - pandas >=1.3.0
+    - pandas >=1.4.0
     - xarray >=0.21.0
     - h5netcdf >=1.0.2
     - packaging
     - typing_extensions >=4.1.0
     - xarray-einstats >=0.3
 
 test:
+  source_files:
+    - arviz
   imports:
     - arviz
   requires:
     - pip
+    - pytest
+    - pytest-cov
+    - cloudpickle
+    - bokeh >=1.4.0
   commands:
     - pip check
+    - pytest arviz/tests --ignore="arviz/tests/external_tests" -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file)"  # [not win]
+    - pytest arviz/tests --ignore="arviz/tests/external_tests" -k "not (test_io_function or test_io_method or test_warning_rc_file or test_bad_rc_file or test_deterministic or test_empty_inference_data_object or test_rctemplate_updated or test_rc_context_file or test_plot_forest_single_value or test_plot_density_discrete_combinedims or test_plot_violin or test_plot_density_float)"  # [win]
```

Comment:
Can the tests fail because of missing a part of the test dependencies:
- https://github.com/arviz-devs/arviz/blob/v0.17.1/requirements-optional.txt
- https://github.com/arviz-devs/arviz/blob/v0.17.1/requirements-external.txt
See https://github.com/arviz-devs/arviz/blob/v0.17.1/requirements-test.txt#L5-L6